linkerd/app/outbound/src/http/concrete.rs

```diff
@@ -158,7 +145,13 @@ impl<N> Outbound<N> {
                 .push_on_service(svc::MapErr::layer_boxed())
                 .lift_new_with_target()
                 .check_new_new_service::<Balance<T>, (SocketAddr, Metadata), http::Request<http::BoxBody>>()
-                .push(http::NewClassifyGateSet::<classify::Response, _, _, _>::layer_via(mk_breaker))
+                .push(http::NewClassifyGateSet::<classify::Response, _, _, _>::layer_via(|target: &Balance<T>| {
+                    breaker::Params {
+                        accrual: target.parent.param(),
+                        // TODO(eliza): should this reuse an existing queue configuration?
+                        channel_capacity: 100,
```

Comment:
What questions do you need to answer to resolve this TODO?

Reply:
Well, my thought process was that IMO it's probably not something that should have its own config knob, as it's kind of an internal implementation detail...but we have to have *some* kind of number for it. We already have a configuration value for channel capacity, but I wasn't totally sure if it made sense to reuse that here. I guess it is a channel of requests (sort of --- it's a channel of classifications produced from the responses to requests, which should be <= the number of requests), so it probably does make sense to use that bound here as well. But, it felt a bit weird, so I wanted to get a second opinion.